sections/footer.liquid

```diff
@@ -72,6 +72,22 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'image' -%}
+                  <div class="footer-block__details-content footer-block-image{% if block.settings.alignment != 'left' %} {{ block.settings.alignment }}{% endif %}">
+                    {%- if block.settings.image != blank -%}
+                      <img
+                        src="{{ block.settings.image | img_url: '400x' }}"
+                        alt="{{ block.settings.image.alt | escape }}"
+                        loading="lazy"
+                        width="{{ block.settings.image.width }}"
+                        height="{{ block.settings.image.height }}"
+                        class=""
+                        style="max-width: min(100%, {{ block.settings.image_width }}px); height: auto;"
```

Comment:
I'm trying to understand the usage of `min` here. Do you know when is the `100%` width used? The max width the image can get is 200px and most mobile device resolution have a minimum of 320px so I'm trying to think in which case the 100% width is used.

Reply:
When the image is part of a row with 4 columns, the width of its column might be < 200px

![Kapture 2021-07-07 at 10 22 41](https://user-images.githubusercontent.com/3710548/124776246-492a2600-df0d-11eb-8c6b-38fb20e4074c.gif)

Reply:
^ exactly. So images don't end up overlapping on certain breakpoints when the container ends up being less than `200px`.

Reply:
Thank you both for the explanation 👍